go/vt/proto/vttest/vttest.pb.go

```diff
@@ -135,9 +135,9 @@ type Keyspace struct {
 	// shards inside this keyspace. Ignored if redirect is set.
 	Shards []*Shard `protobuf:"bytes,2,rep,name=shards,proto3" json:"shards,omitempty"`
 	// number of replica tablets to instantiate. This includes the primary tablet.
-	ReplicaCount int32 `protobuf:"varint,6,opt,name=replica_count,json=replicaCount,proto3" json:"replica_count,omitempty"`
+	ReplicaCount int32 `protobuf:"varint,6,opt,name=replica_count,default=2,json=replicaCount,proto3" json:"replica_count,omitempty"`
 	// number of rdonly tablets to instantiate.
-	RdonlyCount int32 `protobuf:"varint,7,opt,name=rdonly_count,json=rdonlyCount,proto3" json:"rdonly_count,omitempty"`
+	RdonlyCount int32 `protobuf:"varint,7,opt,name=rdonly_count,default=1,json=rdonlyCount,proto3" json:"rdonly_count,omitempty"`
```

Comment:
It is odd that this file is changing but `vttest.proto` has not changed. Re-running `make proto` override this change locally.

Reply:
This is because I manually changed this, and the change is not even valid. Proto3 does not support custom default values, and I'm stuck now trying to think about how to work around this 😬